app.py

```diff
@@ -54,16 +32,39 @@ def experience():
 
 @app.route('/resume/education', methods=['GET', 'POST'])
 def education():
-    '''
-    Handles education requests
-    '''
+    """
+    Handle education requests
+    """
     if request.method == 'GET':
-        return jsonify({})
+        # convert education data to dictionary and return as JSON
+        return jsonify([edu.__dict__ for edu in data['education']])
 
     if request.method == 'POST':
-        return jsonify({})
+        required_fields = ['course', 'school', 'start_date', 'end_date', 'grade', 'logo']
+        if not request.json:
+            return jsonify({'error': 'No data provided'}), 400
+
+        missing_fields = [field for field in required_fields if field not in request.json]
+        if missing_fields:
+            return jsonify({'error': f'Missing required fields: {", ".join(missing_fields)}'}), 400
+
+        # If we used database, it will generate the id for us
+        if data['education']:
+            new_id = max(edu.id for edu in data['education'] if edu.id is not None) + 1
```

Comment:
Can we move this to a separate helper method as this might be useful at quite a few places

Reply:
Sure thing!

Reply:
This requires that the dataclasses have default values. Which makes sense if we were to look at it from a real application perspective. Models usually have default values. Should I change that in the models.py?

Reply:
Or should I leave that to be a future improvement (as other people have yet to merge their changes in)

Reply:
Good point, but let's track it in a separate issue and PR

Reply:
I added the ids. What it does, is that it sets the default value to 1. 
1. This can be fixed by simply adding a db (definitely maybe overboard for this)
2. set ids as type UUID using the uuid library to generate unique id's everytime. 
3. Other than that, if there is no ID, it just sets it as 1 by default.